.ci/redmine_jjb.yaml

```diff
@@ -0,0 +1,79 @@
+- job-template:
+    name: "{jjb_proj}"
+    project-type: freestyle
+    folder: libvma
+    properties:
+        - github:
+             url: "https://github.com/Mellanox/libvma"
+        - build-discarder:
+            days-to-keep: 30
+            num-to-keep: 30
+        - inject:
+            keep-system-variables: true
+            properties-content: |
+              jjb_proj={jjb_proj}
+    description: The job is checking the RM issue. Do NOT edit this job through the Web GUI !
+    concurrent: true
+    parameters:
+        - string:
+            name: "git_repository"
+            default: "ssh://git@github.com/mellanox/libvma"
+            description: "Git repository path"
+
+    wrappers:
+        - workspace-cleanup
+        - build-user-vars
+        - timestamps
+    
+    triggers:
+        - github-pull-request:
+            cron: 'H/5 * * * *'
+            trigger-phrase: '.*\bbot:retest\b.*'
```

Comment:
lets add option to run only this job (like we have in RIvermax):
'.*\bbot:(?:rm:)?retest\b.*'

Reply:
good idea, will add